build/scripts-3.4/pydeface.py

```diff
@@ -0,0 +1,180 @@
+#!/opt/local/bin/python
```

Comment:
Did you mean to add this build folder? I'm confused why there is build and scripts, and both have the same pydeface executable.